test/public/CreateX.computeCreateAddress_2Args.t.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+import {BaseTest} from "../utils/BaseTest.sol";
+import {CreateX} from "../../src/CreateX.sol";
+
+contract CreateX_ComputeCreateAddress_2Args_Public_Test is BaseTest {
+    /*´:°•.°+.*•´.*:˚.°*.˚•´.°:°•.°•.*•´.*:˚.°*.˚•´.°:°•.°+.*•´.*:*/
+    /*                      HELPER VARIABLES                      */
+    /*.•°:°.´+˚.*°.˚:*.´•*.+°.•°:´*.´•*.•°.•°:°.´:•˚°.*°.˚:*.´+°.•*/
+    uint256 internal cachedNonce;
+
+    /*´:°•.°+.*•´.*:˚.°*.˚•´.°:°•.°•.*•´.*:˚.°*.˚•´.°:°•.°+.*•´.*:*/
+    /*                            TESTS                           */
+    /*.•°:°.´+˚.*°.˚:*.´•*.+°.•°:´*.´•*.•°.•°:°.´:•˚°.*°.˚:*.´+°.•*/
+
+    modifier whenTheNonceValueDoesNotExceed18446744073709551614(address deployer, uint64 nonce) {
+        vm.assume(nonce < type(uint64).max);
+        if (deployer.code.length != 0) {
+            vm.assume(nonce != 0);
+        }
+        vm.setNonce(deployer, nonce);
+        _;
+    }
+
+    function testFuzz_WhenTheNonceValueDoesNotExceed18446744073709551614(
+        address deployer,
+        uint64 nonce
+    ) external whenTheNonceValueDoesNotExceed18446744073709551614(deployer, nonce) {
+        vm.startPrank(deployer);
+        address createAddressComputedOnChain = address(new CreateX());
+        // It returns the 20-byte address where a contract will be stored.
+        assertEq(createX.computeCreateAddress(deployer, nonce), createAddressComputedOnChain);
+        vm.stopPrank();
```

Comment:
A nit, but might want to stop prank before the `createX.computeCreateAddress` call, to sanity check that we're not using `msg.sender` in that method

Reply:
Yeah that's a good point - I made all the `stopPrank` statements more specific now so we sanity check this everywhere: https://github.com/pcaversaccio/createx/pull/14/commits/caad20013dd604d041c8d039e262ef8734b165b3.